ui/externs/ui.js

```diff
@@ -395,7 +388,7 @@ shaka.extern.ISeekBar = class {
  * @interface
  * @exportDoc
  */
-shaka.extern.ISeekBar.SeekBarFactory = class {
+shaka.extern.ISeekBarFactory = class {
```

Comment:
Please use the same convention as we did for IUIElement, IUIRangeElement, IUISettingsMenu.  That would be `shaka.extern.IUISeekBar` and `shaka.extern.IUISeekBar.Factory`.  Thanks!

Reply:
Done